src/PaymentGateways/Gateways/Stripe/LegacyStripeAdapter.php

```diff
@@ -20,7 +20,7 @@ public function loadLegacyStripeWebhooksAndFilters()
     {
         $settings = give_get_settings();
         $gatewaysFromSettings = $settings['gateways'] ?? [];
-        $gatewaysFromOption = give_get_option('gateways');
+        $gatewaysFromOption = give_get_option('gateways_v3', []);
```

Comment:
Do we need to add a layer of abstraction for getting the gateways option? This seems to happen in multiple places now.

Reply:
I like that idea, I know we have some legacy functions around this but they do more than get the option and they're being used for different purposes during implementation.  It would be nice to have first class GiveWP settings & options repository.